### ISSUE TITLE
Can I use it without oh-my-zsh?

### ISSUE THREAD

**Issue by wolczek** [arialdomartini/oh-my-git#67]
I use zsh and zprezto because oh-my-zsh is very slow.
Is there a way to install oh-my-git on that env? (no luck so far)

**Comment by stonewareslord** [comment]
Yes. You can use antigen to install oh-my-git. Follow the antigen instructions, but leave out this line: `antigen-use oh-my-zsh`. Then in your zshrc, add `setopt prompt_subst`. Your prompt should now work with oh-my-git!

If you want colors, you should download and source [this](https://github.com/robbyrussell/oh-my-zsh/blob/599831b85a5fd4564ab820530e2bb45f37ca3f99/lib/spectrum.zsh) file or just add the contents to your zshrc.

This appears to work for me, but let me know if it doesn't for you.

**Comment by DYW972** [comment]
Hi 🖖 !

We really need antigen to install Oh-My-Git? 

Can we create it for zsh and add it to the project?

**Comment by arialdomartini** [comment]
Hi!
I think that's technically possible. I should spend some time and make it more independent from antigen. If you have any hints how to proceed, you are more than welcome!

**Comment by DYW972** [comment]
Hi!

Thank you for your answer!

It will be my first time contributing so if I can help I will be happy!

I'm just beginning learning shell and I like omg.

I'm learning how your scripts work in order to make one for `zsh` without using antigen or `omz`.

If you have some advice you are welcome!

🖖

**Comment by DYW972** [comment]
I did it!! I already have working without oh-my-zsh! Was a big journey tho zsh core system but I made it.

Now I'm learning how to contribute on GitHub and then I will propose my update.

Still, If you have some advice you are welcome!

🖖
